hooks/useApprove.ts

```diff
@@ -150,19 +147,28 @@ function useApprove({
           break;
         }
       }
+      track('TX Signed', {
+        operation,
+        hash,
+        method: 'approve',
+        contractName: 'Market',
+        spender,
+        qty: Number(quantity),
```

Comment:
I see below that you are just using the bigint for `qty`, should this also be the case?

Another question, should we consider formatting this numbers with their corresponding decimals?

Reply:
we'll send the values in floating point